src/libtorch.cc

```diff
@@ -897,6 +897,8 @@ ModelInstanceState::ProcessRequests(
        std::to_string(request_count) + " requests")
           .c_str());
 
+  NVTX_RANGE(nvtx_, "ProcessRequests " + Name());
```

Comment:
will these compile if TRITON_ENABLE_NVTX not defined?

Reply:
Ah, nevermind I see it will: https://github.com/triton-inference-server/common/blob/1603d152f2b090194183abe09034f4b9f62fbd49/include/triton/common/nvtx.h#L57